index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/> 
+  <link rel="stylesheet" href="style.css">
+  <title>Mobile Version</title>
+</head>
+<body>
+  <nav>
+    <ul class="nav-bar">
+      <li class="logo">Logo</li>
+      <li><img class="hambug" src="images/Union.png" alt="hamburger"></li>
+    </ul>
+  </nav>
+  <section class="section-one">
+    <h1>Hey There.<br/>
+      I'm Have</h1>
+    <h2>I'm a software Developer</h2>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hestiate to contact me.
+    </p>
+    <ul class="items">
+      <li><img class="item" src="images/Github1.png" alt="Github"></li>
+      <li><img class="item" src="images/Inner1.png" alt="Inner"></li>
+      <li><img class="item" src="images/Peace1.png" alt="Peace"></li>
+      <li><img class="item" src="images/Twitter1.png" alt="Twitter"></li>
+      <li><img class="item" src="images/Mail1.png" alt="mail"></li>
+    </ul>
+  </section>
```

Comment:
# Please match the Figma design:
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/153245009-e2553c56-d465-45f3-b3db-edf27c277bfc.png) | ![template2](https://user-images.githubusercontent.com/24703171/153245080-5d241589-bff6-40fc-a29c-536e3a60e884.PNG) |
- The background image is overlapping with the paragraph text which makes it unreadable.
- Kindly add some spacing between elements